CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing
+
+We're still figuring out how this will look but for now if you need support, please raise an issue and we will get back
+to you within a few days.
+
+# Developing
+
+To compile and run tests
+
+`./gradlew test`
+
+If you're authorized to publish to nexus, which is a process done manually by a developer, you'll need to have been
```

Comment:
Add a step to create a release tag and create a release?

Reply:
oh, right 😬 I think I need to brush up a bit on the process. Maybe we could quickly pair on this later?

Reply:
I was also thinking maybe we wanna keep publishing to sonatype in the build as a manual step. WDYT?

Reply:
Yeah happy to pair. I have some other smaller open source suggestions

Reply:
I never got back to looking at this - @curtiseppel is it ok with you if we merge this and we loop back on doing proper release tags / descriptions at a later date when we have more time to give Kestrel love?